docs/workflow/development.md

```diff
@@ -10,61 +10,52 @@ TODO
+|--------|---------------|----------------------|--------|
+|--------| Developer     | Reviewer             |--------|
 
 **Author:** The person that created this card (issue/pull request). Responsible for answering questions (issues/pull requests) and implement/discuss requested changes (pull requests).
 
 #### Open
 
-The *Open* section contains selected **issues** that are important for the current season. Issues that have a **red** tag should be in here. However, if you want to work on issues that are not in *Open*: Feel free to do so.
+The *Open* section contains by the dev-leads selected **issues** that are important and of high priority for the current iteration. However, if you want to work on issues that are not in *Open*: Feel free to do so.
 
 **Move** (or **add**) an issue into *In Progress* **and** assign yourself when you started working on it.
 
 #### In Progress
 
 The *In Progress* section contains:
-- **issues** that have an assignee, but no open pull request. As soon as there is an open pull request fixing this issue, the issue card should be replaced by this pull request card (removing the Issue from the Project but not closing it as it is not fixed in the master yet). The issue must then be mentioned with the `fixes #Issue_NO` in the pull requests description.
+
+- **issues** that have an assignee, but no open pull request. As soon as there is an open pull request fixing this issue, the issue card should be replaced by this pull request card (removing the Issue from the Project but not closing it as it is not fixed in the main yet). The issue must then be mentioned with the `fixes #Issue_NO` in the pull requests description.
 - **pull requests** that are not ready for review yet and are currently wip.
 
-**Note:** `fixes #Issue_NO` is a keyword on github. Github will automatically close the mentioned issue whenever the corresponding pull request was merged. Do **not** close issues that have not being fixed in the master yet (even if there is a pull request for it)!
+**Note:** `fixes #Issue_NO` is a keyword on github. Github will automatically close the mentioned issue whenever the corresponding pull request was merged. Do **not** close issues that have not being fixed in the main yet (even if there is a pull request for it)!
 
-**Move** a pull request into *Dev Done* when you have finished your work **and** tested the pull request yourself on relevant platforms. At this stage a pull requests' description should be finalized (fill out the template properly).
+**Move** a pull request into *Request for Review* when you have finished your work **and** tested the pull request yourself on relevant platforms. At this stage a pull requests' description should be finalized (fill out the template properly).
 
-#### Dev Done
+#### Request for Review
 
-The *Dev Done* section only contains **pull requests** that are ready for review.
+The *Request for Review* section only contains **pull requests** that are finished™.
```

Comment:
Pull requests do not have to be done to be in ready for review. That was the reason for renaming this column which used to be called "Done".
Because you may want to request feedback on a PR that isn't done yet.
The "done" state should be indicated by whether it is a draft PR or not. Maybe add that to the documentation as well?